README.md

```diff
@@ -26,13 +26,18 @@ OpenTelemetry.
 Each service reads it's configuration from the environment. Specific environment
 variables:
 
-- HONEYCOMB_API_KEY - Your honeycomb API key
-- HONEYCOMB_DATASET - The name of the dataset you want to write to
+- `HONEYCOMB_API_KEY` - Your honeycomb API key
+- `HONEYCOMB_DATASET` - The name of the dataset you want to write to
+- `OTEL_EXPORTER_OTLP_ENDPOINT=https://api.honeycomb.io`
+
 Some services use vanilla OTEL:
-- OTEL_EXPORTER_OTLP_HEADERS='x-honeycomb-team=api-key,x-honeycomb-dataset=greetings'
+
+- `OTEL_EXPORTER_OTLP_HEADERS='x-honeycomb-team=api-key,x-honeycomb-dataset=greetings'`
+
 If using dogfood:
-- OTEL_EXPORTER_OTLP_ENDPOINT=https://api-dogfood.honeycomb.io
-- HONEYCOMB_API_ENDPOINT=https://api-dogfood.honeycomb.io
+
+- `OTEL_EXPORTER_OTLP_ENDPOINT=https://api-dogfood.honeycomb.io`
```

Comment:
I formatted these -- but this is a public-facing document. Do we want `dogfood` mentioned in it at ALL?

Reply:
I can generalize it in another PR